wemake_python_styleguide/violations/naming.py

```diff
@@ -476,3 +478,30 @@ class UnderScoredNumberNameViolation(SimpleViolation):
     #: Error message shown to the user.
     error_template = 'Found underscored name pattern "{0}"'
     code = 114
+
+
+class UpperCaseAttributeViolation(ASTViolation):
+    """
+    Forbids to use anything but snake_case for naming attributes on a class.
+
+    Reasoning:
```

Comment:
Can you please propose a solution for this violation?

Reply:
Okay. :)